src/BuildResult.php

```diff
@@ -40,4 +44,36 @@ public function getErrors(): array
     {
         return $this->errors;
     }
+
+    public function getMetas(): Metas
```

Comment:
Not directly related to this PR, but I suppose that `getMetadata()` would be more correct (which should also be modified in the builder).